### PR TITLE
If pn was not enabled by user, do not show pn switch in IOS

### PR DESCRIPTION
**Pull request by gabrielbazan7**
fix: #4032

### CODE REVIEW

**Comment by bityogi** [comment]
I would suggest (for the sake of consistency) to use: `index.PNEnabledByUser`

`index.usePushNotifications && index.PNEnabledByUser`

I did notice that all methods and properties are on $scope itself, but that also could also be changed to have those methods directly on the controller rather than on $scope.

File: src/js/controllers/preferencesGlobal.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ angular.module('copayApp.controllers').controller('preferencesGlobalController',
       $scope.glideraEnabled = config.glidera.enabled;
       $scope.glideraTestnet = config.glidera.testnet;
       $scope.pushNotifications = config.pushNotifications.enabled;
+      cordova.plugins.diagnostic.isRemoteNotificationsEnabled(function(isEnabled) {
```
Comment (cmgustavo) [inline src/js/controllers/preferencesGlobal.js]: What about WP? If the plugin doesn't support WP, we should check it before

**Comment by cmgustavo** [comment]
I agree with @bityogi about the global variable name, but this solution in general is not Ok. I mean, you are only hiding the option (in config it 's enabled). What about if I want to enable it again?

**Comment by cmgustavo** [comment]
Needs rebase.

**Comment by gabrielbazan7** [comment]
Updated

**Comment by cmgustavo** [comment]
ACK. Tested on iOS.

**Pull request merged**